src/Application/Adapters/WordPress/Preset.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Adapters\WordPress;
+
+use NunoMaduro\PhpInsights\Domain\Contracts\Preset as PresetContract;
+use PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff;
+use SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @internal
+ */
+final class Preset implements PresetContract
+{
+    /**
+     * {@inheritDoc}
+     */
+    public static function getName(): string
+    {
+        return 'wordpress';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function get(): array
+    {
+        return [
+            'exclude' => [
+                'wp-admin',
+                'wp-includes',
+            ],
+            'remove'  => [
+                TypeHintDeclarationSniff::class,
+                \WooCommerce::class,
+            ],
+            'config'  => [
+                ForbiddenFunctionsSniff::class => [
+                    'forbiddenFunctions' => [
+                        'var_dump' => null,
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function shouldBeApplied(array $composer): bool
+    {
+        /** @var string[] $requirements */
+        $requirements = $composer['require'] ?? [];
+
+        if ($requirements) {
+            return self::composerDiscovery($requirements);
+        }
+
+        return self::manualInstallationDiscovery();
+    }
+
+    /**
+     * Defining ways to discover WordPress through composer.
+     *
+     * @param  array  $requirements  Composer requirements list.
```

Comment:
array<string, string>

Reply:
Same as above, how it should be? now is correct? (btw, I don't like to mark resolve conversation when I'm not the reviewer)

Reply:
No. It should be `@param  array<string, string>  $requirements`